Cargo.toml

```diff
@@ -12,4 +12,8 @@ members = [
 ]
 
 [profile.dev]
-opt-level = 2
\ No newline at end of file
+opt-level = 2
+
+[patch.crates-io]
+ecdsa = { git = "https://github.com/RustCrypto/signatures.git" }
+elliptic-curve = { git = "https://github.com/RustCrypto/traits.git" }
```

Comment:
Probably forgot the newline at the end here.
Btw I am using [Taplo](https://crates.io/crates/taplo-cli) in my CIs for a while now, to great success.